webapp/src/main/webapp/news-list-view/js/NewsListService.js

```diff
@@ -51,4 +51,17 @@ export function saveSettings(saveSettingsURL, settings) {
       throw new Error('Response code indicates a server error', resp);
     }
   });
+}
+
+export function illustrationURLResize(illustrationURL, querySelector, idItem, newsListViewName) {
+  if ((idItem === 'articleItem-1' || idItem === 'articleItem-2' || idItem === 'articleItem-3')
+  && newsListViewName === 'newsLatest') {
+    return illustrationURL.concat('&size=107x107').toString();
+  } else {
```

Comment:
Why, for this case, it is not possible to get element.height, and element.width ?

Reply:
It is possible,but we can get it from the CSS file .